packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -139,162 +140,165 @@ Use `shorebird flutter versions list` to list available versions.
       flutterRevisionForRelease = revision;
     }
 
-    final originalFlutterRevision = shorebirdEnv.flutterRevision;
-    final switchFlutterRevision =
-        flutterRevisionForRelease != originalFlutterRevision;
-
-    if (switchFlutterRevision) {
-      await shorebirdFlutter.useRevision(revision: flutterRevisionForRelease);
-    }
-
-    final flutterVersionString = await shorebirdFlutter.getVersionAndRevision();
-
-    final buildProgress = logger.progress(
-      'Building release with Flutter $flutterVersionString',
```

Comment:
It was nice to have this log framed as `"Building with Flutter <revision>"` instead of `installing Flutter` since we don't always need to install a revision of Flutter right? Most times, the revision they use will be default version that is already installed. I think it would be nice to keep the old copy wdyt?

Reply:
We print both "installing" and "building", the diff is just bad.